dashboard/database.py

```diff
@@ -0,0 +1,135 @@
+"""Functions for interacting with the database."""
+
+from os import environ as ENV
+
+from psycopg import connect, Connection
+from dotenv import load_dotenv
+import streamlit as st
+import pandas as pd
+
+
+@st.cache_resource
+def get_connection() -> Connection:
+    """gets a connection"""
+    connection = connect(
+        port=ENV["DB_PORT"],
+        dbname=ENV["DB_NAME"],
+        host=ENV["DB_ENDPOINT"],
+        user=ENV["DB_USER"],
+        password=ENV["DB_PASSWORD"]
+    )
+    return connection
```

Comment:
This could just be:

```suggestion
    return connect(
        port=ENV["DB_PORT"],
        dbname=ENV["DB_NAME"],
        host=ENV["DB_ENDPOINT"],
        user=ENV["DB_USER"],
        password=ENV["DB_PASSWORD"]
    )
```